core/controllers/community_dashboard.py

```diff
@@ -28,18 +28,6 @@
 import utils
 
 
-class CommunityDashboardPage(base.BaseHandler):
```

Comment:
Just curious—Once this is removed, how does the routing to the community dashboard happen?

Reply:
@sagangwee, as this handler has `open_access` ACL and it was just serving an HTML file, we are going to the server it statically (like the way we server /contact, /about etc. pages)
Check app_dev.yml file changes to see how this will be served statically.